packages/instrumentation-remix/src/instrumentation.ts

```diff
@@ -426,7 +426,7 @@ export class RemixInstrumentation extends InstrumentationBase {
                   const formData = await clonedRequest.formData();
                   const { actionFormDataAttributes: actionFormAttributes } = plugin.getConfig();
                   formData.forEach((value, key) => {
-                    if (actionFormAttributes[key] !== false) {
+                    if (actionFormAttributes[key] && actionFormAttributes[key] !== false) {
```

Comment:
any chance we should do a check for falsy values instead of direct check against false?
```suggestion
                    if (actionFormAttributes[key] && !!actionFormAttributes[key] !== false) {
```

Reply:
nevermind, ignore.